ddtrace/ext/errors.py

```diff
@@ -7,19 +7,18 @@
 from ddtrace.constants import ERROR_MSG
 from ddtrace.constants import ERROR_STACK
 from ddtrace.constants import ERROR_TYPE
-from ddtrace.internal.utils.deprecation import deprecated
-from ddtrace.internal.utils.deprecation import deprecation
+
+from ..vendor.debtcollector.removals import remove
+from ..vendor.debtcollector.removals import removed_module
 
 
 __all__ = [ERROR_MSG, ERROR_TYPE, ERROR_STACK]
 
-deprecation(
-    name="ddtrace.ext.errors",
-    message=(
-        "Use `ddtrace.constants` module instead. "
-        "Shorthand error constants will be removed. Use ERROR_MSG, ERROR_TYPE, and ERROR_STACK instead."
-    ),
-    version="1.0.0",
+removed_module(
+    module="ddtrace.ext.errors",
+    replacement="ddtrace.constants",
+    message="Use ERROR_MSG, ERROR_TYPE, and ERROR_STACK instead.",
+    removal_version="1.0.0",
```

Comment:
how does this get rendered?

Reply:
The ddtrace.ext.errors module usage is deprecated and will be removed in version `1.0.0`:  Use ERROR_MSG, ERROR_TYPE, and ERROR_STACK instead ., please use ddtrace.constants instead

Reply:
I will need to remove the period at the end of the message

Reply:
🤔 that message is definitely a little weird.

I wonder if we just get rid of the custom `message`, what do you think?

hopefully the only thing people are using from `ddtrace.ext.errors` is `ERROR_*` constants, so should be easy to move to `ddtrace.constants.ERROR_*`

What do you think?